src/pages/RealmScans/style.less

```diff
@@ -0,0 +1,21 @@
+@import './../pages';
+
+@activeEp: #e333cf52;
+@textColor: inherit;
+@inputBackground: inherit;
+
+#AddMal {
+  background: inherit !important;
+}
+
+#malp span.MalLogin {
+  background: inherit;
+}
+
+#malp span#MalData {
+  background: inherit;
+}
+
+#malp {
+  background: inherit;
+}
```

Comment:
Chapter list highlighting only works in the dark theme.
`.mal-sync-active`
![image](https://user-images.githubusercontent.com/12599334/170266965-3a22e7c3-abcf-4cde-89c8-5c8437093017.png)